clients/src/main/java/org/apache/kafka/clients/admin/OffsetSpec.java

```diff
@@ -73,18 +73,20 @@ public static OffsetSpec maxTimestamp() {
     }
 
     /**
-     * Used to retrieve the offset with the local log start offset,
-     * log start offset is the offset of a log above which reads are guaranteed to be served
-     * from the disk of the leader broker, when Tiered Storage is not enabled, it behaves the same
-     * as the earliest timestamp
+     * Used to retrieve the local log start offset.
+     * Local log start offset is the offset of a log above which reads
+     * are guaranteed to be served from the disk of the leader broker.
+     * <br/>
+     * Note: When tiered Storage is not enabled, it behaves the same as retrieving the earliest timestamp offset.
      */
     public static OffsetSpec earliestLocalSpec() {
         return new EarliestLocalSpec();
     }
 
     /**
-     * Used to retrieve the offset with the highest offset of data stored in remote storage,
-     * and when Tiered Storage is not enabled, we won't return any offset (i.e. Unknown offset)
+     * Used to retrieve the highest offset of data stored in remote storage.
+     * <br/>
+     * Note: When tiered storage is not enabled, we will return unknown offset.
      */
     public static OffsetSpec latestTierSpec() {
         return new LatestTierSpec();
```

Comment:
This is not related to this PR, but KIP-1005 (https://cwiki.apache.org/confluence/display/KAFKA/KIP-1005%3A+Expose+EarliestLocalOffset+and+TieredOffset) calls it `LatestTieredSpec` rather than `LatestTierSpec`

ping @FrankYang0529 as you are the author of #16781

Reply:
Thanks for finding this. Yes, we should follow the KIP. @brandboat, could you fix it in this PR? I'm afraid if I create another PR may block your progress.

Reply:
Everything has been addressed, thanks everyone.